api-standards/structure.md

```diff
@@ -0,0 +1,84 @@
+An extension of the [API Standards](../API_STANDARDS.md), with implementation details specific to development of RESTful APIs for ONS.
+
+## RESTful API Structure and Syntax
+
+### Syntax & types
+
+  * Field names snake case, e.g. `total_count`
+  * Use appropriate types for fields, e.g. numbers should be ints not strings
```

Comment:
could you add a difference between 10 and 10.0 pls

Reply:
Yeah we should probably update this example

```suggestion
  * Use appropriate types for fields, e.g. numbers should be numeric types for the language in use and not strings, so integers, floating point numbers etc.
```

Reply:
DIdn't refresh and see nathans comment before posting. I'll add a 'it is acceptable to use a string for a field which may contain a number or other string, but use the appropriate type where its consistent'

Reply:
So JSON can handle floating numbers and Open API 3.0 defines the type number as either integers or floating points - https://swagger.io/specification/v3/#:~:text=The%20formats%20defined%20by%20the%20OAS%20are%3A

So in the above example numbers can be integers or floating points. It might be better to just link off to the Open API documentation?